browser/brave_stats/first_run_util.cc

```diff
@@ -37,4 +45,16 @@ base::Time GetFirstRunTime(PrefService* local_state) {
 #endif  // #BUILDFLAG(IS_ANDROID)
 }
 
+bool IsFirstRun([[maybe_unused]] PrefService* local_state) {
+#if BUILDFLAG(IS_ANDROID)
+  if (g_is_first_run) {
+    return true;
+  }
+  GetFirstRunTime(local_state);
```

Comment:
I think it would be better to break out the setter/getter as this reads strange because it doe not actually get anything.